app/controllers/api/console.php

```diff
@@ -38,3 +39,59 @@
 
         $response->dynamic($variables, Response::MODEL_CONSOLE_VARIABLES);
     });
+
+
+App::post('/v1/console/assistant')
+    ->desc('Ask Query')
+    ->groups(['api', 'assistant'])
+    ->label('scope', 'public')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION])
+    ->label('sdk.namespace', 'assistant')
+    ->label('sdk.method', 'chat')
+    ->label('sdk.description', '/docs/references/assistant/chat.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_TEXT)
+    ->label('abuse-limit', 15)
```

Comment:
Is the default time interval 1 hour?

Reply:
yes https://github.com/appwrite/appwrite/blob/fcd4a69ad1b3db67c6d89f02c5f230c2038a4b35/app/controllers/shared/api.php#L121